{
  "modified": "2024-10-16T14:49:28Z",
  "published": "2024-10-16T14:49:28Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-10144",
  "summary": "Malicious code in pytrv (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "pytrv",
        "purl": "pkg:pypi/pytrv"
      },
      "versions": [
        "0.0.1"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-09082",
        "import_time": "2024-10-24T00:57:07.364185962Z",
        "modified_time": "2024-10-16T14:49:28Z",
        "sha256": "fe65455693843ccf42009ceaa1e37766afa42a50f75c4c0294a33c5694bd9c6a",
        "source": "reversing-labs",
        "versions": [
          "0.0.1"
        ]
      }
    ]
  }
}
